content/app/content_main_runner_impl.cc

```diff
@@ -661,8 +661,6 @@ int ContentMainRunnerImpl::Initialize(const ContentMainParams& params) {
   base::CommandLine::ForCurrentProcess()->AppendSwitch(switches::kNoZygote);
   base::CommandLine::ForCurrentProcess()->AppendSwitch(
       switches::kInProcessGPU);
-  base::CommandLine::ForCurrentProcess()->AppendSwitch(
-      switches::kDisableGpuCompositing);
```

Comment:
Can you remove this change, because without waitsync if we enable gpu compositing, there may be stability issues.
You can enable this change  together with the waitsync changes. Currently you can remove this.
https://github.com/Samsung/Castanets/pull/281/commits/556fa512d1a57808aaf5180c093838124904c33e
https://github.com/Samsung/Castanets/pull/281/commits/1642e65fe6459d843e137cca51b664829892e98e